pybv/io.py

```diff
@@ -29,7 +29,7 @@
 
 def write_brainvision(data, sfreq, ch_names, fname_base, folder_out,
                       events=None, resolution=1e-7, scale_data=True,
-                      fmt='binary_float32', meas_date=None):
+                      fmt='binary_float32', meas_date=None, unit='µV'):
```

Comment:
look at the "parameters" section in the docstring as well:

```
    data : ndarray, shape (n_channels, n_times)
        The raw data to export. Data is assumed to be in
        **volts**. The data will be stored in **microvolts**.

```

That will need to be edited as well.

Reply:
OK, that wasn't even true before the change, I'll update it.